src/components/chatbox/message.tsx

```diff
@@ -216,20 +179,21 @@ const AssistantMessage = ({
 	)
 }
 
-// TODO(kevinwang): error styling.
 const ApplicationMessage = ({ text }: { text: string }) => {
 	return (
 		<div className={classNames(s.message, s.assistant)}>
-			<IconTile className={classNames(s.iconError)}>
+			<IconTile className={s.strong}>
 				<IconWand24 style={{ width: 24, height: 24 }} />
 			</IconTile>
-			<Text /* Body/200/Medium */
-				size={200}
-				weight="medium"
-				className={classNames(s.content)}
-			>
-				{text}
-			</Text>
+			<div className={s.applicationMessage}>
+				<IconAlertDiamondFill24
+					className={s.critical}
+					style={{ width: 14, height: 14, marginTop: 2, marginBottom: 2 }}
+				/>
+				<Text /* Body/100/Regular */ size={100} weight="regular">
+					{text}
+				</Text>
+			</div>
```

Comment:
Added markup implements the error icon style in

[✓ Global: 429- adjust the language and red icon that displays for this state. ](https://app.asana.com/0/0/1205523765248052/f)